src/libraries/System.Drawing.Common/src/System/Drawing/Drawing2D/GraphicsPath.Unix.cs

```diff
@@ -692,7 +692,7 @@ public void Flatten(Matrix matrix)
             Flatten(matrix, FlatnessDefault);
```

Comment:
Cannot write on row 626-653:  For all overloads of `public void AddString(string s, FontFamily family, int style, float emSize, PointF origin, StringFormat format)` method `format` is nullable